exist-core/src/main/java/org/exist/xmlrpc/RpcAPI.java

```diff
@@ -926,7 +927,7 @@ String hasUserLock(String path)
 
     List<String> getGroups() throws EXistException, PermissionDeniedException;
 
-    List<List> getIndexedElements(String collectionName, boolean inclusive)
+    List<List<Object>> getIndexedElements(String collectionName, boolean inclusive)
```

Comment:
this works? 
<img width="448" alt="image" src="https://github.com/eXist-db/exist/assets/1700062/005b9241-b01e-4b8d-8792-538505226f4c">